en/releases/centreon-core.md

```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.04.20
+
+#### Bug fixes
+
+- [Configuration] Fixed reset untouched values using massive changes on hosts
+- [Custom Views] Fixed sharing of views with LDAP groups
+- [Downtime] Fixed the adding of a downtime on a resource from the French GUI
+
+#### Security fixes
+
+- CSRF - delete any command
+- CSRF - delete any poller
+- CSRF - delete any host
+- CSRF - delete any SNMP trap
+- CSRF - delete any service
+- CSRF - delete any contactGroup
+- CSRF vulnerability allowing to delete any user	
+- CSRF vulnerability allowing to delete many kinds of objects
+
```

Comment:
These tickets are missing in french and english version
- MON-11656 : Error in Event view
- MON-11139 : [Dependency to upgrade on : centreon/centreon] - composer prod. bump nelmio/cors-bundle from 1.5.6 to 2.1.1
- MON-11154 : [Dependency to upgrade on : centreon/centreon] - npm prod. bump axios from 0.21.1 to 0.21.3

Reply:
MON-11656 should not appear in the RN as the issue has been hit during regression test phasis

Reply:
@lpinsivy  should MON-11139 and MON-11154 be added to the RN (dependency upgraded for security purpose)